tests/tests.js

```diff
@@ -68,6 +68,22 @@ describe("htmlbars-inline-precompile", function() {
       assert.equal(transformed, "var compiled = Ember.HTMLBars.template(precompiled(hello));", "tagged template is replaced");
     });
 
+    it("moduleName (filenameRelative) is passed through to precompiler", function() {
+      var code = "import hbs from 'htmlbars-inline-precompile'; var compiled = hbs('hello');";
+
+      var precompile = function(template, options) {
+        assert.equal(options.moduleName, 'module-name-test.js');
+
+        return "precompiled(" + template + ")";
+      };
+
+      babel.transform(code, {
+        blacklist: ['strict', 'es6.modules'],
+        plugins: [HTMLBarsInlinePrecompile(precompile)],
+        filenameRelative: 'module-name-test.js'
```

Comment:
Sorry to be such a complainer here, but I am currently not sure if the current test fully covers what the `moduleName` functionality does. Do you think there is a way we can use `babel.transformFileSync` or something, so it tests that the `file.opts.filenameRelative` is the correct value? I just want to make sure that if babel is transforming templates from a file, then we are reading the correct value. Using `filenameRelative` looks ok, but at the moment it looks to me like we are only stubbing the property and not ensuring that it is really the name of a file.